packages/styles/scss/patterns/sub-patterns/link-list/_link-list.scss

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+@import '../../../globals/utils/hang';
+
+@mixin link-list {
+  .#{$prefix}--link-list {
+    &__heading {
+      margin-top: $carbon--spacing-07;
+      margin-bottom: $carbon--spacing-05;
+
+      @include type-style('expressive-heading-02');
+
+      color: $text-01;
+    }
+    &__CTA {
+      .#{$prefix}--card {
+        &__wrapper {
+          min-height: 116px;
+        }
+        &:not(:hover) {
+          background: $ui-background;
+        }
+        &__heading {
+          @include carbon--type-style('body-short-02');
+
+          color: $link-01;
+        }
+        svg {
+          fill: $interactive-04;
+        }
+      }
+    }
+    &__list {
+      padding-bottom: $carbon--layout-01;
```

Comment:
What's your thoughts around adding this? I'm not seeing it in the design spec.

Reply:
done